src/TConsole.cpp

```diff
@@ -231,7 +217,7 @@ TConsole::TConsole(Host* pH, ConsoleType type, QWidget* parent)
     centralLayout->setContentsMargins(0, 0, 0, 0);
 
     if (mType == MainConsole) {
-        mpCommandLine = new TCommandLine(pH, mpCommandLine->MainCommandLine, this, mpMainDisplay);
+        mpCommandLine = new TCommandLine(pH, QLatin1String("main"), TCommandLine::MainCommandLine, this, mpMainDisplay);
```

Comment:
```suggestion
        mpCommandLine = new TCommandLine(pH, qsl("main"), TCommandLine::MainCommandLine, this, mpMainDisplay);
```

qsl here?

Reply:
Not sure of this, because there are so many instances of it in the whole application - OTOH there does seem to only be one in THIS compilation unit so perhaps `qsl(...)` is tolerable in **this** case.

Reply:
Done.